packages/core/src/App/Containers/RealAccountSignup/finished-set-currency.jsx

```diff
@@ -5,15 +5,24 @@ import { isDesktop, isMobile } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
 
 const FinishedSetCurrency = ({ current, onCancel, onSubmit, prev }) => {
-    const IconPrevCurrency = () => <Icon icon={`IcCurrency-${prev.toLowerCase()}`} height={120} width={90} />;
+    const IconPrevCurrency = () =>
+        prev ? <Icon icon={`IcCurrency-${prev.toLowerCase()}`} height={120} width={90} /> : null;
     const IconNextCurrency = () => <Icon icon={`IcCurrency-${current.toLowerCase()}`} height={120} width={90} />;
-    const IconArrow = () => <Icon icon='IcArrowPointerRight' color='red' width={50} height={20} />;
+    const IconArrow = () => (prev ? <Icon icon='IcArrowPointerRight' color='red' width={50} height={20} /> : null);
 
     const IconWon = ({ className }) => <Icon className={className} icon='IcCheckmarkCircle' color='green' />;
 
+    const message = prev
```

Comment:
this won't be caught by the translation script - as it only reads every file and collects the strings inside `localize()` or `<Localize />`  - (no compilation involved).

each of these could be a e.g. a `<Localize />` component instead.